samples/apps/copilot-chat-app/README.md

```diff
@@ -0,0 +1,77 @@
+    2.	Update `appsettings.json` with these settings:
+
+          *	Under the `“ServicePort”` block, make the following configuration changes to match your instance:
+
+            * `“AIService”: “AzureOpenAI”`, or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-davinci-003”,` or whichever option is appropriate for your instance.  
+            * `“Endpoint”:` “Your Endpoint address, i.e. http://project.region.azure.com”
+            * `“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code” 
+
+        * Under the `“EmbeddingConfig”` block, make sure the following configuration changes to match your instance:
+            * `“AIService”: “AzureOpenAI”,` or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-embedding-ada-002”,` or whichever option is appropriate for your instance.    
+            *	`“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code” 
+            
+4. Build the back-end server by following these instructions:
+    1.	In the terminal navigate to  `\samples\apps\copilot-chat-app\SKWebApi`
+    2.	Execute the command `dotnet build`
+    3.	Once the build is complete, Execute the command `dotnet run`
+    4.	Test the back-end server to confirm it is running.
+        * Open a web browser, and navigate to `https://localhost:40443/probe`
+        * You should see a confirmation message: `Semantic Kernel service is up and running`
+>Note: you may need to accept the locally signed certificate on your machine in order to see this message.  It is important to do this, as your browser may need to accept the certificate before allowing the WebApp to communicate with the backend.
+
+>Note: You may need to acknowledge the Windows Defender Firewall, and allow the app to communicate over private or public netowrks as appropriate.
+
+
+5.	Now that the back-end server is setup, and confirmed operating, let’s proceed with setting up the front-end WebApp.
+    1. Navigate to `\apps\copilot-chat-app\webapp`
+    2.	Copy `.env.example` into a new file with the name “`.env`” and make the following configuration changes to match your instance:
+    3. Use the Application (client) ID from the Azure Portal steps above and paste the GUID into the .env file next to `REACT_APP_GRAPH_CLIENT_ID= `
+    4.	Execute the command `Yarn Install`
+    5.	Execute the command `Yarn Start`
+    
+    6. Wait for the startup to complete.
+    7. With the back end and front end running, your web browser should automatically launch and navigate to `https://localhost:3000`
+    8. Sign in in with your account details.
+    9. Grant permission to use your account details.
+    10.	If it fails to load, you may need to validate the certificate of the backend server.  If you see an error message, or if the machine appears to stop, waiting for the connection to the backend server, please see the troubleshooting section to accept your machines certificates.
```

Comment:
Let's pair this down to something like "If you experience any errors or issues, consult the troubleshooting section below."

Reply:
done